test/gui/shared/scripts/helpers/api/webdav_helper.py

```diff
@@ -31,3 +32,15 @@ def resource_exists(user, resource):
 def get_file_content(user, resource):
     response = request.get(get_resource_path(user, resource), user=user)
     return response.text
+
+
+def get_folder_item_number(user, folder_name):
+    path = get_resource_path(user, folder_name)
+    xml_response = request.propfind(path, user=user)
+    total_items = 0
+    root_element = ET.fromstring(xml_response.content)
+    for response_element in root_element:
+        for href_element in response_element:
+            if href_element.tag == '{DAV:}href':
+                total_items += 1
+    return str(total_items - 1)
```

Comment:
Out of curiosity, why do we return `total_items - 1`?

Reply:
because first element is the folder root itself
https://github.com/owncloud/client/pull/10967#discussion_r1243502836